doc/STYLE_GUIDE.md

```diff
@@ -19,39 +18,37 @@
   * Use gender-neutral pronouns and gender-neutral plural nouns.
     * OK: _they_, _their_, _them_, _folks_, _people_, _developers_
     * NOT OK: _his_, _hers_, _him_, _her_, _guys_, _dudes_
-* When combining wrapping elements (parentheses and quotes), terminal
-  punctuation should be placed:
+* When combining wrapping elements (parentheses and quotes), place terminal
+  punctuation:
   * Inside the wrapping element if the wrapping element contains a complete
-    clause — a subject, verb, and an object.
+    clause.
   * Outside of the wrapping element if the wrapping element contains only a
     fragment of a clause.
 * Documents must start with a level-one heading.
 * Prefer affixing links to inlining links — prefer `[a link][]` to
   `[a link](http://example.com)`.
-* When documenting APIs, note the version the API was introduced in at
-  the end of the section. If an API has been deprecated, also note the first
-  version that the API appeared deprecated in.
-* When using dashes, use [Em dashes][] ("—" or `Option+Shift+"-"` on macOS)
-  surrounded by spaces, as per [The New York Times Manual of Style and Usage][].
+* When documenting APIs, update the YAML comment associated with the API as
+  appropriate. This is especially true when introducing or deprecating an API.
+* Use [Em dashes][] ("—" or `Option+Shift+"-"` on macOS) surrounded by spaces,
+  as per [The New York Times Manual of Style and Usage][].
 * Including assets:
   * If you wish to add an illustration or full program, add it to the
     appropriate sub-directory in the `assets/` dir.
-  * Link to it like so: `[Asset](/assets/{subdir}/{filename})` for file-based
-    assets, and `![Asset](/assets/{subdir}/{filename})` for image-based assets.
-  * For illustrations, prefer SVG to other assets. When SVG is not feasible,
-    please keep a close eye on the filesize of the asset you're introducing.
+  * Link to non-image assets: `[Asset](/assets/{subdir}/{filename})`
+  * Link to image assets: `![Asset](/assets/{subdir}/{filename})`
+  * For illustrations, prefer SVG to other assets. SVG or otherwise, please keep
+    a close eye on the filesize of the asset you're introducing.
```

Comment:
Do we have any such asset links? I don't recall coming across any images in the docs that are in this repository.

Reply:
> Do we have any such asset links? I don't recall coming across any images in the docs that are in this repository.

@richardlau `grep -rl "(/assets/" doc` turns up only the style guide so I guess these lines can be removed!

Reply:
Looked for svg in the doc directory too and got nothing. All the assets stuff can be removed from the style guide.